### DISCUSSION TITLE
[DONE] Неполное отображение таблицы

### DISCUSSION THREAD

**Discussion by smplseven** [totumonline/totum-ru-issues-and-discussions#248] [Есть идея, а что если...]
При загрузке таблицы, в которых число строк превышает кол-во на экране, при скролле страницы вниз грузятся оставшиеся строки. Вероятно это сделано для того, чтобы не грузить систему разовой отдачей большого объема информации. Возможно ли увеличить это количество, чтобы грузился большой объем строк или так сделана сама пагинация и механизм неизменный?

![image](https://user-images.githubusercontent.com/113604604/197765847-e5f4de17-96c5-419c-9f60-5de4180ba2d5.png)

**Comment by ttm-alexeypolunin** [comment]
Через некоторое время в PRO можно будет указать для таблицы — какое количество строк подгружать помимо тех, которые помещаются в экран.

**Comment by ttm-alexeypolunin** [comment]
В версии PRO v.3 — параметр таблицы `row_buffering`.

Задает сколько срок подгружать вверх и вниз.

Те если указано 300, то помимо экрана будет загружено еще 600 строк.

**Reply by smplseven** [reply]
Супер,большое спасибо!
